### PR TITLE
Fix get_s unit test

### LINKED ISSUE #22: Fix unit test: utils: `get_s`
At some point in development the function `define_s` was refactored. During this, it was renamed `get_s`. Its test was not refactored. This was not spotted since the unit test was not updated to call `get_s` instead of `define_s`. The unit test needs to be updated so that it correctly calls and tests the refactored version of `get_s`.

### PR DESCRIPTION
**Pull request by grthomson**
## Description
 
This pr introduces a unit test for the function get_s contained in utils/utils.py

closes #22 

## Type of change

*You can delete options that are not relevant.*

- [ ] Bug fix - *non-breaking change*
- [x] New feature - *non-breaking change*
- [ ] Breaking change - *backwards incompatible change, changes expected behaviour*
- [ ] Non-user facing change, structural change, dev functionality, docs ...

## Checklist:

- [x] I have performed a self-review of my own code.
- [x] I have commented my code appropriately, focusing on explaining my design decisions (explain why, not how).
- [x] I have made corresponding changes to the documentation (comments, docstring, etc.. )
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have updated the change log.

<br>

#  Peer review
Any new code includes all the following:

- **Documentation**: docstrings, comments have been added/ updated.
- **Style guidelines**: New code conforms to the project's contribution guidelines.
- **Functionality**: The code works as expected, handles expected edge cases and exceptions are handled appropriately.
- **Complexity**: The code is not overly complex, logic has been split into appropriately sized functions, etc..
- **Test coverage**: Unit tests cover essential functions for a reasonable range of inputs and conditions. Added and existing tests pass on my machine.

### Review comments
Suggestions should be tailored to the code that you are reviewing. Provide context.
Be critical and clear, but not mean. Ask questions and set actions.
<details><summary>These might include:</summary>

- bugs that need fixing (does it work as expected? and does it work with other code
  that it is likely to interact with?)
- alternative methods (could it be written more efficiently or with more clarity?)
- documentation improvements (does the documentation reflect how the code actually works?)
- additional tests that should be implemented
  - Do the tests effectively assure that it
  works correctly? Are there additional edge cases/ negative tests to be considered?
- code style improvements (could the code be written more clearly?)
</details>
<br>

*Further reading: [code review best practices](https://best-practice-and-impact.github.io/qa-of-code-guidance/peer_review.html)*

### CODE REVIEW

File: tests/utils/test_utils.py [hunk 1]
```diff
@@ -226,39 +226,46 @@ def test_get_input_variables():
...
+        "df1_path": "path_to_df_1",
+        "df2_path": "path_to_df_2",
+    }
 
+    expected_output = {
+        "spark_session_size": "small",
+        "df1_path": "path_to_df_1",
+        "df2_path": "path_to_df_2",
+        "s": 9,
+    }
```
Comment (mary-cleaton) [inline tests/utils/test_utils.py]: These could be simplified to just contain the key:value pairs that are relevant to the test, e.g.:

```suggestion
    test_input_variables = {}

    expected_output = {"s": 9}
```

Comment (grthomson) [reply tests/utils/test_utils.py]: Agreed, extra detail doesn't really add anything and is not in keeping with the other tests.

Am going to test GitHub's "Commit suggestion" for this one.

**Review by mary-cleaton — APPROVED** [review]
Requested change has been implemented. No further changes required.
